components/SetupWrapper.tsx

```diff
@@ -39,7 +39,7 @@ export default function SetupWrapper({ children }: SetupWrapperProps) {
             position: 'absolute',
           }}
         >
-          {ApiUrl}
+          {/*{ApiUrl}*/}
```

Comment:
Why this? In testing I thought it was pretty useful seeing the api url used

Reply:
Fairs, I guess I got annoyed with the text and/or removed it for demo purposes and totally forgot it